### PR TITLE
Add page on data discoverability

### PR DESCRIPTION
**Pull request by lauportell**
GDoc:
https://docs.google.com/document/d/1XSTDFLpBZxoBv8xOp6ZzUzEVhb2ux30Enut_w-fYP08/edit?usp=sharing

### CODE REVIEW

File: pages/your_tasks/data_discoverability.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: Data Discoverability
+description: How to make data discoverable
+contributors: [Aina Jené Cortada, Laura Portell Silva]
```
Comment (bedroesb) [inline pages/your_tasks/data_discoverability.md:4]: Do you have more info about Aina Jené Cortada for in the CONTRIBUTORS file?

File: pages/your_tasks/data_discoverability.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+page_id: data_discoverability
+related_pages: 
+  your_domain: human_data
+  data_life_cycle: reusing, sharing
+  your_task: sensitive
```
Comment (bedroesb) [inline pages/your_tasks/data_discoverability.md:9]:
```suggestion
```

We only allow your tasks pages to be linked to Tool assembly pages.

File: pages/your_tasks/data_discoverability.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  {% include image.html file="beacon-api.JPG" inline=true caption="Figure 1. Beacon API funcionality ([Source](https://docs.genomebeacons.org/implementations-options/))" alt="Beacon API" max-width="30em"%}
+
+* {% tool "beacon-ri" %}: Beacon v2 Reference Implementation, B2RI, is an out-of-the-box example implementation of the Beacon v2 protocol. It is an open-source toolkit based on Python programming language and consists of tools for loading metadata, e.g.phenotypic data, from a CSV file and genomic variants from a VCF file into a MongoDB database. It also features the Beacon query engine (REST API) and comes bundled with an example dataset (CINECA synthetic cohort EUROPE UK1) comprising synthetic data.
+
+  {% include image.html file="beacon-ri.JPG" inline=true caption="Figure 2. Beacon RI funcionality. ([Source](https://docs.genomebeacons.org/implementations-options/))" alt="Beacon RI" max-width="30em" %}
+
+Documentation: https://b2ri-documentation.readthedocs.io/en/latest/ 
+GitHub Repository: https://github.com/EGA-archive/beacon2-ri-tools-v2
```
Comment (bedroesb) [inline pages/your_tasks/data_discoverability.md:77]: Don't know if I like the way this is listed. Any other opinions?

File: pages/your_tasks/data_discoverability.md (hunk 1 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md:4]: sure! I thought she was included already

Comment (lauportell) [reply pages/your_tasks/data_discoverability.md:4]: done!

File: pages/your_tasks/data_discoverability.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+* Detailed Metadata: Provide comprehensive metadata for your datasets, including titles, descriptions, keywords, creators, dates, and other relevant information.
+* Documentation: Include thorough documentation that explains the dataset, its structure, how it was collected, processed, and any limitations.
+* Standard Schemas: Use standardized metadata schemas to ensure consistency and interoperability.
+* Standard Formats: Use widely accepted data formats to ensure compatibility and ease of use.
+* Public Repositories: Deposit your data in reputable public data repositories that are indexed by search engines and widely used by the research community.
+
+### Solutions
+
+* There are many appropriate tools for detailed metadata and data documentation during the project. Check the [Documentation and metadata page](metadata_management) for more information.
```
Comment (Nazeeefa) [inline pages/your_tasks/data_discoverability.md:26]: You mean "There are several appropriate tools to create detailed (or comprehensive) metadata and document data properly for the project (not just during?) - check out the [Documentation...."

File: pages/your_tasks/data_discoverability.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+## How can you discover controlled access data?
+ 
+### Description
+
+Discovering research data for re-analysis can occur at different levels of granularity. Initially, researchers browse online catalogues that describe studies, datasets, related publications, variables, and some data distributions. This basic discovery may suffice if the datasets meet all the criteria. However, to find dataset that meet specific combinations of attributes, such as “adults diagnosed with COVID-19 in the last year, fully vaccinated, with no underlying health conditions,” researchers must either contact the authors or request data access and verify themselves. This process is feasible for a small number of datasets and cooperative data controllers but it is usually time-consuming and uncertain. To streamline this, data discovery at the source allows users to query data non-disclosively, determining its relevance before requesting full access.
```
Comment (Nazeeefa) [inline pages/your_tasks/data_discoverability.md:40]: It says "such as “adults diagnosed with COVID-19 in the last year, fully vaccinated, with no underlying health conditions," but it's not clear if this is an example of a reference to an article?

File: pages/your_tasks/data_discoverability.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+### Considerations
```
Comment (Nazeeefa) [inline pages/your_tasks/data_discoverability.md:42]: It may be useful to link to relevant RDMkit pages here, for example, you can mention links to pages such as Ethical aspects, GDPR Compliance, etc in 'Ethical and Legal Compliance' part. Similarly, Data sensitivity page can be mentioned in 'Privacy and Security measures' part. Unless all these relevant pages are going to be mentioned under Related Pages.

**Review by Nazeeefa — CHANGES_REQUESTED** [review]

File: pages/your_tasks/data_discoverability.md (hunk 5 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md:40]: it is only an example - how would you rephrase it to make it clear?

File: pages/your_tasks/data_discoverability.md (hunk 6 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md:42]: done!

File: pages/your_tasks/data_discoverability.md (hunk 4 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md]: changed the text a bit to clarify

**Comment by bedroesb** [comment]
@Nazeeefa Could you have a look if your comments are covered and approve if so? thanks!

File: pages/your_tasks/data_discoverability.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+* Detailed Metadata: ensure comprehensive metadata for your datasets, including detailed descriptions of studies, datasets, variables, and any available distributions.
+* Data Catalogs and Repositories: use well-maintained online catalogs and repositories that support controlled access data, and check for advanced search features to filter datasets by specific attributes.
+* Data Access Policies: get familiar with the data access policies of different repositories and datasets, understanding the requirements and procedures for requesting access to controlled data.
+* Ethical and Legal Compliance: ensure compliance with ethical guidelines and legal regulations governing data use and sharing, and obtain necessary approvals from institutional review boards or ethics committees if required. Check the [GDPR compliance](gpdr_compliance) and [Ethical aspects](ethics) pages for more information.
```
Comment (Nazeeefa) [inline pages/your_tasks/data_discoverability.md:47]: @lauportell Just a typo in page linking - can you fix it to gdpr_compliance? Thanks!

File: pages/your_tasks/data_discoverability.md (hunk 3 above)
Comment (Nazeeefa) [reply pages/your_tasks/data_discoverability.md:75]: Yeah it's hard to tell what are the two ways - I think what might help is to add to the text what is the documentation and GitHub repo for? e.g. simply "Documentation for B2RI: ". What do  you think, @lauportell?

File: pages/your_tasks/data_discoverability.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+Beacon v2 is organised in two main blocks: the Beacon Framework and the Beacon Model. 
+The Framework defines the format for the requests and responses, whereas the Model defines the structure of the biological data response.
+
+This dual-system approach not only broadens the scope for diverse Models – using different domains such as images, pathogens, or infectious diseases – but also reinforces the adaptability of the Framework. The overall function of these components is to provide the instructions to design a REST API that could be implemented as a stand-alone product or, preferably, extending existing data management solutions. 
+
+Consequently, the 'beaconised' data represents a significant enhancement in data discoverability with minimal risks. 
+
+Currently, there are two ways to implement a Beacon:
+
+* On top of existing tools (API): targeted to those organizations equipped with well-organised and structured data housed in databases, whether SQL or NoSQL, and possess the necessary resources and expertise to interpret and implement the Beacon v2 specification and construct an API on top of an existing tool.
```
Comment (Nazeeefa) [inline pages/your_tasks/data_discoverability.md:66]: Add "This is" before "targeted to...." and, revise heading to e.g. "API on top of existing tools" or something better (not sure why API is written in parentheses).

File: pages/your_tasks/data_discoverability.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+The latest version, {% tool "beacon-v2" %}, introduced expanded query options, enabling the retrieval of biological or technical (meta)data through filters defined via CURIEs. This includes, but not limited to, parameters such as phenotypes, disease codes, sex, or age, providing researchers with a nuanced approach to data inquiries.
+
+Beacon v2 is organised in two main blocks: the Beacon Framework and the Beacon Model. 
+The Framework defines the format for the requests and responses, whereas the Model defines the structure of the biological data response.
+
+This dual-system approach not only broadens the scope for diverse Models – using different domains such as images, pathogens, or infectious diseases – but also reinforces the adaptability of the Framework. The overall function of these components is to provide the instructions to design a REST API that could be implemented as a stand-alone product or, preferably, extending existing data management solutions. 
+
+Consequently, the 'beaconised' data represents a significant enhancement in data discoverability with minimal risks. 
+
+Currently, there are two ways to implement a Beacon:
```
Comment (Nazeeefa) [inline pages/your_tasks/data_discoverability.md:64]: "Currently, there are two..." can be placed next to the previous sentence after "...minimal risks."

File: pages/your_tasks/data_discoverability.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+
+## How can you discover controlled access data?
+ 
+### Description
+
+Discovering research data for re-analysis can occur at different levels of granularity. Initially, researchers browse online catalogues that describe studies, datasets, related publications, variables, and some data distributions. This basic discovery may suffice if the datasets meet all the criteria. However, to find dataset that meet specific combinations of attributes — such as identifying datasets with particular combinations of attributes, like 'adults diagnosed with COVID-19 in the last year, fully vaccinated, with no underlying health conditions' (as an example) — researchers must either contact the authors or request data access and verify themselves. This process is feasible for a small number of datasets and cooperative data controllers but it is usually time-consuming and uncertain. To streamline this, data discovery at the source allows users to query data non-disclosively, determining its relevance before requesting full access.
```
Comment (Nazeeefa) [inline pages/your_tasks/data_discoverability.md:40]: remove "(as an example)" and replace like (written after "combinations of attributes,") with "for example".

**Review by Nazeeefa — CHANGES_REQUESTED** [review]

File: pages/your_tasks/data_discoverability.md (hunk 3 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md]: done

File: pages/your_tasks/data_discoverability.md (hunk 7 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md]: done

File: pages/your_tasks/data_discoverability.md (hunk 8 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md]: done

File: pages/your_tasks/data_discoverability.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+  {% include image.html file="beacon-ri.JPG" inline=true caption="Figure 2. Beacon RI functionality. ([Source](https://docs.genomebeacons.org/implementations-options/))" alt="Beacon RI" max-width="30em" %}
+
+Documentation for B2RI: https://b2ri-documentation.readthedocs.io/en/latest/ 
+GitHub Repository for Beacon v2: https://github.com/EGA-archive/beacon2-ri-tools-v2 
+
+Building on B2RI, the {% tool "b4omop" %} (B4OMOP) software allows for the integration of a Beacon onto any OMOP Common Data Model (CDM) database. This enables organizations using the OMOP CDM to leverage the Beacon framework for querying and sharing genomic and phenotypic data.
+
+Documentation: https://gitlab.bsc.es/impact-data/impd-beacon_omopcdm
```
Comment (bedroesb) [inline pages/your_tasks/data_discoverability.md:77]: is this documentation url still needed if it is part of the tool snippet?

File: pages/your_tasks/data_discoverability.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+
+  {% include image.html file="beacon-api.JPG" inline=true caption="Figure 1. Beacon API functionality ([Source](https://docs.genomebeacons.org/implementations-options/))" alt="Beacon API" max-width="30em"%}
+
+* {% tool "beacon-ri" %}: Beacon v2 Reference Implementation, B2RI, is an out-of-the-box example implementation of the Beacon v2 protocol. It is an open-source toolkit based on Python programming language and consists of tools for loading metadata, e.g. phenotypic data, from a CSV file and genomic variants from a VCF file into a MongoDB database. It also features the Beacon query engine (REST API) and comes bundled with an example dataset (CINECA synthetic cohort EUROPE UK1) comprising synthetic data.
+
+  {% include image.html file="beacon-ri.JPG" inline=true caption="Figure 2. Beacon RI functionality. ([Source](https://docs.genomebeacons.org/implementations-options/))" alt="Beacon RI" max-width="30em" %}
+
+Documentation for B2RI: https://b2ri-documentation.readthedocs.io/en/latest/
```
Comment (bedroesb) [inline pages/your_tasks/data_discoverability.md:70]: I would also take here the url away that is already in the tool snippet. Regarding the Documentation, it should be more clearly linked from the GitHub repo in my opinion. than this one can also be taken away here.

Comment (bedroesb) [reply pages/your_tasks/data_discoverability.md:70]: image
The docs url can be added to the about of the repo, which is best practice

File: pages/your_tasks/data_discoverability.md (hunk 11 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md]: removed!

File: pages/your_tasks/data_discoverability.md (hunk 12 above)
Comment (lauportell) [reply pages/your_tasks/data_discoverability.md]: done

**Comment by bedroesb** [comment]
@Nazeeefa If you could give it a final look, you can do the merge!

**Comment by Nazeeefa** [comment]
Thank you @lauportell for the great work! ✨ and thanks @bedroesb for the support!

**Review by Nazeeefa — APPROVED** [review]
